backend/back/src/webhooks/services/gihub-webhook.service.ts

```diff
@@ -3,7 +3,7 @@ import { GrpcService } from "../../grpc/grpc.service";
 import { createHmac, timingSafeEqual } from "crypto";
 import { ConfigService } from "@nestjs/config";
 
-type GithubActions = "opened";
+type GithubActions = "opened" | "reopened" | "closed";
```

Comment:
Is "GithubActions" a good name ? Could GithubIssueState be a better name ? It's a type (not an array), action is not representing the underlying value and its the state of something, not of GitHub globally

Reply:
Done

Reply:
(named it GithubBodyActions as the key is name "action" and is used in all Github payloads: issues, pr, commits...)